field_utils/mapl_udunits2status.F90

```diff
@@ -0,0 +1,28 @@
+! Status values for udunits2 procedures
+! The values are the same as the udunits2 utStatus C enum
+module mapl_udunits2status
+
+   implicit none
+
+   enum, bind(c)
+       enumerator :: &
+       UT_SUCCESS = 0, & ! Success
+       UT_BAD_ARG, & ! An argument violates the function's contract
+       UT_EXISTS, & ! Unit, prefix, or identifier already exists
+       UT_NO_UNIT, & ! No such unit exists
+       UT_OS, & ! Operating-system error. See "errno".
```

Comment:
Documenting here is of course quite useful for the user.

Reply:
Yes. In this case, I think that bringing over the comments is helpful.